tools/IndexRebuilder/IndexRebuilder.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>Microsoft.Health.Fhir.IndexRebuilder</RootNamespace>
```

Comment:
Is this an internal application i.e. not mean to publish/consumption? If so can we prefix with `Microsoft.Health.Internal`

Reply:
It is an internal tool. Please change.